File: app/src/main/java/com/senerunosoft/puantablosu/model/SingleScore.java
package com.senerunosoft.puantablosu.model;

public class SingleScore {
    private String playerId;
    private int score;

    public SingleScore() {
    }

    public SingleScore(String playerName, int score) {
        this.playerId = playerName;
        this.score = score;
    }

    public String playerId() {
        return playerId;
    }

    public void playerId(String playerName) {
        this.playerId = playerName;
    }

    public int getScore() {
        return score;
    }

    public void setScore(int score) {
        this.score = score;
    }
}

File: app/src/main/java/com/senerunosoft/puantablosu/model/SingleScore.kt
package com.senerunosoft.puantablosu.model

data class SingleScore(
    var playerId: String = "",
    var score: Int = 0
) {
    // Empty constructor for Gson
    constructor() : this("", 0)

    // Getter methods to match original Java API (for backward compatibility)
    fun playerId(): String = playerId
    fun setPlayerId(playerId: String) {
        this.playerId = playerId
    }
    fun getScore(): Int = score
    fun setScore(score: Int) {
        this.score = score
    }
}